pkg/eventcounter/api/api_test.go

```diff
@@ -1735,7 +1870,26 @@ func getDate(t time.Time) int64 {
 	return time.Date(t.Year(), t.Month(), t.Day(), 0, 0, 0, 0, jpLocation).Unix()
 }
 
-func newEventCounterService(t *testing.T, mockController *gomock.Controller) *eventCounterService {
+func newEventCounterService(t *testing.T, mockController *gomock.Controller, SpecifiedEnvironmentId *string, SpecifiedOrgRole *accountproto.AccountV2_Role_Organization, SpecifiedEnvRole *accountproto.AccountV2_Role_Environment) *eventCounterService {
```

Comment:
```suggestion
func newEventCounterService(t *testing.T, mockController *gomock.Controller, specifiedEnvironmentId *string, specifiedOrgRole *accountproto.AccountV2_Role_Organization, SpecifiedEnvRole *accountproto.AccountV2_Role_Environment) *eventCounterService {
```

Reply:
Thank you for review!
I fixed with https://github.com/bucketeer-io/bucketeer/pull/871/commits/d4c7d9f6ea6bf70be0c7baf7dad9cc4918d0ccd7 .